### ISSUE TITLE
New scenarios thread

### ISSUE THREAD

**Issue by Grelo** [07th-mod/higurashi-patch-compiler#33]
So here it is.

Onikakushi:

![scenario_oni](https://user-images.githubusercontent.com/22790067/36641082-aa9b333a-1a2a-11e8-8f47-5f521f0be6b1.png)
![scenario_oni2](https://user-images.githubusercontent.com/22790067/36641083-aab78dbe-1a2a-11e8-8635-1c5308352ae8.png)
![scenario_oni3](https://user-images.githubusercontent.com/22790067/36641084-aad46b64-1a2a-11e8-9ca0-208a1597b552.png)

Watanagashi:

![scenario_wata](https://user-images.githubusercontent.com/22790067/36641117-f8b46e38-1a2a-11e8-8a0a-2d68e795a60b.png)
![scenario_wata2](https://user-images.githubusercontent.com/22790067/36641118-f8cfd006-1a2a-11e8-823c-52364beb8dd9.png)
![scenario_wata3](https://user-images.githubusercontent.com/22790067/36641119-f8e75456-1a2a-11e8-952a-6f5bc75d4224.png)

Tatarigoroshi:

![scenario_tata](https://user-images.githubusercontent.com/22790067/36641125-07dd8f2a-1a2b-11e8-9e15-5df8187a1e8b.png)
![scenario_tata2](https://user-images.githubusercontent.com/22790067/36641126-07f6a47e-1a2b-11e8-80bf-98a88bcc1563.png)
![scenario_tata3](https://user-images.githubusercontent.com/22790067/36641127-080e70d6-1a2b-11e8-9472-7648873d85bb.png)

Himatsubushi:

![scenario_hima](https://user-images.githubusercontent.com/22790067/36641135-2b8e64ee-1a2b-11e8-9c3b-3413678982a5.png)
![scenario_hima2](https://user-images.githubusercontent.com/22790067/36641136-2ba77790-1a2b-11e8-8c24-62bea4815089.png)
![scenario_hima3](https://user-images.githubusercontent.com/22790067/36641137-2bc06be2-1a2b-11e8-91df-016650e8fca3.png)

Meakashi:

![scenario_meak](https://user-images.githubusercontent.com/22790067/36641144-37130b6c-1a2b-11e8-878d-4c7b948cb68f.png)
![scenario_meak2](https://user-images.githubusercontent.com/22790067/36641145-372af70e-1a2b-11e8-8ae0-25654539b3b0.png)
![scenario_meak3](https://user-images.githubusercontent.com/22790067/36641146-3743ae70-1a2b-11e8-8915-31e38ab35496.png)

**Comment by InochiPM** [comment]
Here is my versions of the scenario files, along with the PSD below. 
I also went ahead and made the three for Rei, and the other two for Kai since it seems like MG is using the old Kai scenario translations for the new files. 

PSD File: https://app.box.com/s/ofjztsb2xivpywn8ojzbnzjuge2uq2t1

Higurashi: 
Onikakushi-hen:
![scenario_a](https://user-images.githubusercontent.com/13459080/36772086-db7f5960-1c21-11e8-8782-94ee51b9c2ad.png)
![scenario_b_j](https://user-images.githubusercontent.com/13459080/36772087-db8cc88e-1c21-11e8-9e63-4c61bf286d4b.png)
![scenario_b](https://user-images.githubusercontent.com/13459080/36772088-db9adbae-1c21-11e8-96ea-c14f1f72c94b.png)
![scenario_c_j](https://user-images.githubusercontent.com/13459080/36772090-dba7bf68-1c21-11e8-913c-d9dd7754e20e.png)
![scenario_c](https://user-images.githubusercontent.com/13459080/36772091-dbb5160e-1c21-11e8-989b-25b4e2def2d6.png)

Watanagashi-hen:
![scenario_a](https://user-images.githubusercontent.com/13459080/36772112-ef2f57c6-1c21-11e8-9aad-e0bc1acb1460.png)
![scenario_b_j](https://user-images.githubusercontent.com/13459080/36772113-ef3e2df0-1c21-11e8-92e1-c5cb8e160f60.png)
![scenario_b](https://user-images.githubusercontent.com/13459080/36772114-ef4f3794-1c21-11e8-92e3-6760d9d6e08a.png)
![scenario_c_j](https://user-images.githubusercontent.com/13459080/36772115-ef5d1e40-1c21-11e8-81b5-e85c994cf20c.png)
![scenario_c](https://user-images.githubusercontent.com/13459080/36772116-ef6af57e-1c21-11e8-8325-18066db86a7c.png)

Tatarigoroshi-hen:
![scenario_a](https://user-images.githubusercontent.com/13459080/36772139-0df105ec-1c22-11e8-8b33-a555965bac19.png)
![scenario_b_j](https://user-images.githubusercontent.com/13459080/36772141-0e003e9a-1c22-11e8-9449-54126698b624.png)
![scenario_b](https://user-images.githubusercontent.com/13459080/36772142-0e0ddf50-1c22-11e8-97d4-471a28263a07.png)
![scenario_c_alt](https://user-images.githubusercontent.com/13459080/36772143-0e1b5c48-1c22-11e8-8e3c-5ed978f2c8e8.png)
![scenario_c_j](https://user-images.githubusercontent.com/13459080/36772144-0e2909e2-1c22-11e8-91d4-f80d9fd70362.png)
![scenario_c](https://user-images.githubusercontent.com/13459080/36772145-0e36b506-1c22-11e8-9324-3d0a67732dcc.png)

Himatsubushi-hen:
![scenario_a](https://user-images.githubusercontent.com/13459080/36772161-20f02cc2-1c22-11e8-8f6e-d7e36c7fcf02.png)
![scenario_b_j](https://user-images.githubusercontent.com/13459080/36772162-20ffd258-1c22-11e8-8225-bac6643b5ca2.png)
![scenario_b](https://user-images.githubusercontent.com/13459080/36772163-210e1cbe-1c22-11e8-92c0-e6b44365c876.png)
![scenario_c_j](https://user-images.githubusercontent.com/13459080/36772164-211c8f92-1c22-11e8-932d-7ccdcdc10f97.png)
![scenario_c](https://user-images.githubusercontent.com/13459080/36772165-212e3ec2-1c22-11e8-8a14-5e92b7356b7d.png)

Higurashi Kai:
Meakashi-hen: 
![scenario_a](https://user-images.githubusercontent.com/13459080/36772178-318d3070-1c22-11e8-8ace-7a5218650b57.png)
![scenario_b_j](https://user-images.githubusercontent.com/13459080/36772179-319d048c-1c22-11e8-8c34-80a67f2ee1c2.png)
![scenario_b](https://user-images.githubusercontent.com/13459080/36772180-31ab8c5a-1c22-11e8-8472-8c7a5edbbe28.png)
![scenario_c_j](https://user-images.githubusercontent.com/13459080/36772181-31b94322-1c22-11e8-8161-6d3ce645eabc.png)
![scenario_c](https://user-images.githubusercontent.com/13459080/36772182-31c8e0de-1c22-11e8-9ded-1c152ba6f947.png)

Tsumihoroboshi-hen: 
![scenario_a](https://user-images.githubusercontent.com/13459080/36772197-4584eeec-1c22-11e8-8507-b220ccb6ae5e.png)
![scenario_b_j](https://user-images.githubusercontent.com/13459080/36772198-459490f4-1c22-11e8-8252-a0dda263384b.png)
![scenario_b](https://user-images.githubusercontent.com/13459080/36772199-45a2ccc8-1c22-11e8-95e4-b208137d14c6.png)
![scenario_c_j](https://user-images.githubusercontent.com/13459080/36772200-45b26ee4-1c22-11e8-9024-4d7a8b1498e9.png)
![scenario_c](https://user-images.githubusercontent.com/13459080/36772201-45c0effa-1c22-11e8-86bc-36f20e377671.png)

Minagoroshi-hen:
![scenario_a](https://user-images.githubusercontent.com/13459080/36772246-71e31edc-1c22-11e8-87ff-7d2f7ac7d0e5.png)
![scenario_b_j](https://user-images.githubusercontent.com/13459080/36772247-71f452e2-1c22-11e8-9712-b0aa4af84282.png)
![scenario_b](https://user-images.githubusercontent.com/13459080/36772248-72046d1c-1c22-11e8-87a4-3244fff4ca39.png)
![scenario_c_j](https://user-images.githubusercontent.com/13459080/36772249-72133c5c-1c22-11e8-8172-a235b48dc529.png)
![scenario_c](https://user-images.githubusercontent.com/13459080/36772250-72237568-1c22-11e8-89a0-76aba55fe02a.png)

Matsuribayashi-hen: 
![scenario_a](https://user-images.githubusercontent.com/13459080/36772260-819f5f34-1c22-11e8-9bea-b1ab88a2d025.png)
![scenario_b_j](https://user-images.githubusercontent.com/13459080/36772261-81aed518-1c22-11e8-8980-ed712352601e.png)
![scenario_b](https://user-images.githubusercontent.com/13459080/36772262-81bdea44-1c22-11e8-9be3-2629f00f85e1.png)
![scenario_c_j](https://user-images.githubusercontent.com/13459080/36772296-a85c88d6-1c22-11e8-8054-708a24984b3a.png)
![scenario_c](https://user-images.githubusercontent.com/13459080/36772264-81df1cbe-1c22-11e8-8188-520e589514c4.png)

Higurashi Rei:
Saikoroshi-hen:
![scenario_a](https://user-images.githubusercontent.com/13459080/36772313-c86d8666-1c22-11e8-84fe-3349879fb9bd.png)
![scenario_b_j](https://user-images.githubusercontent.com/13459080/36772314-c87e6922-1c22-11e8-89d1-9f2afc40ec30.png)
![scenario_b](https://user-images.githubusercontent.com/13459080/36772315-c88e478e-1c22-11e8-8c15-2f00fdc0e75b.png)
![scenario_c_j](https://user-images.githubusercontent.com/13459080/36772317-c89f344a-1c22-11e8-9c59-becc8ba5bc47.png)
![scenario_c](https://user-images.githubusercontent.com/13459080/36772318-c8af5cf8-1c22-11e8-9fc3-9d14d543045c.png)

Batsukoishi-hen:
![scenario_a](https://user-images.githubusercontent.com/13459080/36772328-d67a449c-1c22-11e8-8a13-4139ad01e81d.png)
![scenario_b_j](https://user-images.githubusercontent.com/13459080/36772329-d689b9fe-1c22-11e8-866a-6a42e670f21b.png)
![scenario_b](https://user-images.githubusercontent.com/13459080/36772331-d6987b24-1c22-11e8-9c83-5b098a002f1d.png)
![scenario_c_j](https://user-images.githubusercontent.com/13459080/36772332-d6a99e7c-1c22-11e8-8d52-4f3e024ad954.png)
![scenario_c](https://user-images.githubusercontent.com/13459080/36772333-d6b87d48-1c22-11e8-9b3a-004a4b8cc89b.png)

Hirukowashi-hen:
![scenario_a](https://user-images.githubusercontent.com/13459080/36772344-e234c424-1c22-11e8-8557-a0510f68bace.png)
![scenario_b_j](https://user-images.githubusercontent.com/13459080/36772345-e24a3a70-1c22-11e8-9ec8-3e63136af09e.png)
![scenario_b](https://user-images.githubusercontent.com/13459080/36772346-e25da088-1c22-11e8-88ce-2d87cc1412aa.png)
![scenario_c_j](https://user-images.githubusercontent.com/13459080/36772347-e27764aa-1c22-11e8-999c-e1286631f322.png)
![scenario_c](https://user-images.githubusercontent.com/13459080/36772348-e287013a-1c22-11e8-9437-429e2996d96e.png)

**Comment by enumag** [comment]
Sorry guys, I kinda forgot to give you my feedback here.

I have to admit that I like @Grelo's images way more in pretty much all aspects - fonts, subtitles and backgrounds. The only exception is Hima where I like the shrine background more.

The images from Minagoroshi are not bad but none of them strikes me as "ideal" like the bridge for Tata or the scrapyard for Tsumi.

What do you think @ItaloKnox.

**Comment by ItaloKnox** [comment]
100% agreed. I prefer @Grelo's images, but it's not really a big margin. I just think they blend nicely, kinda like they were there from the beginning.

**Comment by Petsnew** [comment]
Yep, I also prefer @Grelo's backgrounds better, with the exception of Himatsubushi and maybe Watanagashi. His titles are great, too. Although I'd say his text font choice isn't the best; it's kinda... weird I guess? Can't really explain why I dislike it.

About @InochiPM's, I think the Minagoroshi image isn't a very good representation of the chapter. The idea behind the Matsuribayashi one is great, not so much the image itself, which is kinda bland. An open sky and/or field really fits the arc. Hmm, Saikoroshi is fine, maybe an image of Rika's house would be better though. And the other two arcs, I think more light-hearted images fit them more, since they're mostly fanservice.

**Comment by Grelo** [comment]
Changed Hima to a new version.

**Comment by enumag** [comment]
@Grelo Do you think you could make scenario images for Someutushi, Kageboushi and Tsukiotoshi? It would be nice for the next release of console arcs.

**Comment by DoctorDiablo** [comment]
@Grelo Here are the images I'd like to use for the console arcs scenario images and the text to go with them:

@Norgus Are you interested in translating this text to Japanese?  I'm not confident enough in my writing skills to attempt it.  If not, these can be English-only.

----------------------------------------------------------------
Someutsushi
-Stain Following-

This chapter can be read after completing Chapter 3: Tatarigoroshi.

Natsumi Kimiyoshi moved away from Okinomiya one year ago
and is seeking a happy life in Kakiuchi City,
but her ties to Hinamizawa inescapably follow her.
![kimi_n1_02](https://user-images.githubusercontent.com/20390474/39496437-69204920-4d6c-11e8-8033-3a7d32df8e2f.png)

----------------------------------------------------------------

Kageboushi
-Spinning shadows-

This chapter is the answer arc for Someutsushi.
It can be read after completing Chapter 5: Meakashi.

Akasaka travels to Kakiuchi City to investigate the
mysteries of Hinamizawa along with Ooishi and Tomoe.
His investigation leads him to Natsumi Kimiyoshi.
![kou_par](https://user-images.githubusercontent.com/20390474/39496456-82732320-4d6c-11e8-816c-45970a089f3f.png)

-----------------------------------------------------------------

Tsukiotoshi
-Exorcism-

This chapter is an alternate take on Tatarigoroshi.
It can be read after completion of Chapter 3: Tatarigoroshi.

Keiichi and friends work together to help Satoko
after her uncle returns to Hinamizawa once again.
![ho1_01](https://user-images.githubusercontent.com/20390474/39496480-9bfb3abc-4d6c-11e8-8b9a-3e102ea7a318.png)

**Comment by Norgus** [comment]
@DoctorDiablo I could give it a go later, but is there not already Japanese for those lines? I mean, where did the English come from?

**Comment by DoctorDiablo** [comment]
Nope, I wrote those.  Sui doesn't provide any descriptions for each chapter, but I added them to be more consistent with the PC release.  I tried to provide a non-spoilery explanation of what each arc is and when they can be read, so people don't have to go reading through wikis to figure all that out.

**Comment by enumag** [comment]
@Norgus Thank you!

@DoctorDiablo @ItaloKnox Wanna check the translation before I get Grelo / Inochi to make the images?

**Comment by ItaloKnox** [comment]
It's Norgus we're talking about, I have no issues with it. Looks great.

**Comment by enumag** [comment]
Thought so. :-)

@Grelo This should be everything you need to make the images. Could you create them?

**Comment by Grelo** [comment]
Sorry was not here for a while and sure. When i finish i will create issue
or post them here

Dňa nedeľa 13. mája 2018 Jáchym Toušek <notifications@github.com>
napísal(a):

**Comment by Grelo** [comment]
@Norgus I need the translation of titles as well. Someutsushi, stain following etc.

**Comment by Norgus** [comment]
@Grelo the "subtitles" used here seem to simply be translations of the titles, so I don't really think it makes sense to paraphrase the title as a subtitle.

I'll look at wikipedia and google a bit to see if subtitles exist.

**Comment by enumag** [comment]
cc @DoctorDiablo

**Comment by DoctorDiablo** [comment]
That's right, the subtitles are just translations of the titles.  We can leave those out in the Japanese version.
